app/app.py

```diff
@@ -448,16 +482,88 @@ def show_thumbs(
     with thumbs_down_button:
         st.button(
             "👎",
-            on_click=post_question_response_to_metric_service,
+            on_click=post_user_rating_feedback_to_metric_service,
             args=(
-                metric_service_endpoint,
+                f"{metric_service_endpoint}/user_feedback",
                 "thumbs_down",
                 question,
                 full_response,
             ),
         )
 
 
+def create_score_threshold_collector(
+    metric_service_endpoint: str,
+    readability_scores: Dict[str, Dict[str, Union[str, float]]],
+) -> None:
+    """Create the readability score threshold collector.
+
+    This function will display a message saying that we have detected the readability score of the response is below a certain threshold.
+    If the user agree to share data with us, we will send this data to our metric service and store them in the metric database.
+
+    Args:
+        metric_service_endpoint (str): the metric service endpoint
+        readability_scores (Dict[str, Dict[str, Union[str, float]]]): a dictionary containing the score, question response.
+    """
+    readability_score_threshold = 55.0
```

Comment:
Nit: I'd have this as a static variable up at the top of here (easier to configure)